docs/reference-guide/modules/events/pages/event-processors/streaming.adoc

```diff
@@ -922,12 +695,12 @@ Alternatively, inject the `EventProcessingConfigurer`, which allows more fine-gr
 public class AxonConfig {
     // omitting other configuration methods...
     @Bean
-    public ConfigurerModule tokenStoreConfigurerModule() {
+    public ConfigurationEnhancer tokenStoreConfigurationEnhancer() {
         TokenStore tokenStore = JdbcTokenStore.builder()
                                               // …
                                               .build();
 
-        return configurer -> configurer.eventProcessing(
+        return registry -> MessagingConfigurer.with(registry).eventProcessing(
```

Comment:
Yet again a wrong code sample using the non-existing `MessagingConfigurer#with` method.